satip.c

```diff
@@ -87,7 +87,10 @@ const char *cPluginSatip::CommandLineHelp(void)
          "                                define hard-coded SAT>IP server(s)\n"
          "  -D, --detach                  set the detached mode on\n"
          "  -S, --single                  set the single model server mode on\n"
-         "  -n, --noquirks                disable all the server quirks\n";
+         "  -n, --noquirks                disable all the server quirks\n"
+         "  -r, --portrange=<start>       set a range of ports used for the rt[c]p server\n"
```

Comment:
Hmm, the range is defined by the base port and the number of devices given by the "-d" parameter.  I wouldn't there give a range.
The user should ensure the ports are available, as with every server application?